raiden/tests/fixtures/raiden_network.py

```diff
@@ -125,4 +125,12 @@ def raiden_network(
 
     _raiden_cleanup(request, raiden_apps)
 
+    # The block_number is primed on the app creation, but after the app is
+    # created all the channels are deployed, for the tester implementation this
+    # will advance the block_number with synchronous execution, making the
```

Comment:
> will advance the block_number with synchronous execution

Question for clarification:

So if we deploy all contracts during tests with the tester blockchain, the blockchain's block number will be advanced with each deployment without giving a chance to the alarm task to fire and as such the raiden app would fall behind.

So for that reason you add the `poll_for_new_block()` here?